Update InteractableCard and LinkCard to support the new corner palette (#3223)
jpmorganchase/salt-ds: .changeset/spicy-vans-whisper.md

---
"@salt-ds/core": patch
---

> Added missing link to corner palette in InteractableCard and LinkCard

Reviewer: ```suggestion
Updated InteractableCard and LinkCard to support the new corner palette
```
